operator/cmd/mesh/install.go

```diff
@@ -233,7 +240,7 @@ func Install(kubeClient kube.CLIClient, rootArgs *RootArgs, iArgs *InstallArgs,
 	}
 
 	// Post-install message
-	if profile == "ambient" {
+	if profile == ambientProfile {
 		p.Println("The ambient profile has been installed successfully, enjoy Istio without sidecars!")
```

Comment:
```suggestion
		p.Println("The ambient profile has been installed successfully. Enjoy Istio without sidecars!")
```

(Or better yet remove the whole thing - this will save me another PR!)